### PR TITLE
For #37 test for RsWithBody

### PR DESCRIPTION
**Pull request by olenagerasimova**
Created test for `RsWithBody` and removed todo.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#143](https://github.com/artipie/http/issues/143) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/a8b70977-69ea-4d41-8144-56c80a922658); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: pom.xml [hunk 1]
```diff
@@ -128,4 +128,83 @@ SOFTWARE.
       <version>4.3.6</version>
     </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.jacoco</groupId>
```
Comment (g4s8) [inline pom.xml:134]: @olenagerasimova maybe we can move this plugin configuration to `pluginManagement` of parent pom?

Comment (olenagerasimova) [reply pom.xml:134]: @g4s8I like the idea and I'll submit PR to parent pom. Configuration (limits) then will be set up in each project individually

File: src/test/java/com/artipie/http/rs/RsWithBodyTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ * @since 0.9
+ */
+final class RsWithBodyTest {
+
+    @Test
+    void createsOkWithBody() {
+        final byte[] body = "abc".getBytes();
+        MatcherAssert.assertThat(
+            new RsWithBody(ByteBuffer.wrap(body)),
+            new AllOf<>(
```
Comment (Vatavuk) [inline src/test/java/com/artipie/http/rs/RsWithBodyTest.java:47]: @olenagerasimova this can be simplified to:
```java
Matchers.allOf(
  new RsHasBody(body.getBytes()),
  new RsHasStatus(RsStatus.OK)
)
```

File: src/test/java/com/artipie/http/rs/RsWithBodyTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            )
+        );
+    }
+
+    @Test
+    void appendsBody() {
+        final String body = "def";
+        MatcherAssert.assertThat(
+            new RsWithBody(new RsWithStatus(RsStatus.CREATED), body, StandardCharsets.UTF_8),
+            new AllOf<>(
```
Comment (Vatavuk) [inline src/test/java/com/artipie/http/rs/RsWithBodyTest.java:61]: @olenagerasimova same as in previous comment

File: src/test/java/com/artipie/http/rs/RsWithBodyTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link RsWithBody}.
+ * @since 0.9
+ */
+final class RsWithBodyTest {
+
+    @Test
+    void createsOkWithBody() {
```
Comment (Vatavuk) [inline src/test/java/com/artipie/http/rs/RsWithBodyTest.java:43]: @olenagerasimova test name is not clear, can you rephrase it

**Comment by olenagerasimova** [comment]
@Vatavuk thanks, corrected

**Comment by 0crat** [comment]
Job [#143](https://github.com/artipie/http/issues/143) is already in scope

**Comment by 0crat** [comment]
GitHub issue is still open, won't close

**Comment by g4s8** [comment]
@olenagerasimova please check failing xcop linter which is ok in master branch: https://github.com/artipie/http/pull/143/checks?check_run_id=655537157

**Comment by olenagerasimova** [comment]
@g4s8 corrected

**Comment by Vatavuk** [comment]
@g4s8 please remove this issue from the scope

**Comment by 0crat** [comment]
@vatavuk[/z](https://www.0crat.com/u/vatavuk) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good
